.gitlab/ci.yml

```diff
@@ -100,13 +100,13 @@ tarball suse-42.1:
         # show GLIBC verison
         - *show-glibc-version
 
-tarball ubuntu-16.04:
+tarball ubuntu-18.04:
```

Comment:
Tarballs were explicitly created for supporting older OS versions to be used internally in Ericsson.
I know for sure that we are using `suse-42.1`, I am not sure about `ubuntu-16.04`.

If we are using this tarball, we shall not update it. If we are not using it, maybe just dropping it would be fine.
I recommend to not modify the`.gitlab/ci.yml` file for now.